pkg/broker/dynamicplans/plan.go

```diff
@@ -74,3 +76,29 @@ func (p Plan) String() string {
 
 	return string(s)
 }
+
+func (p *Plan) UnmarshalJSON(data []byte) error {
+	type plan Plan
+
+	var pl plan
+	err := json.Unmarshal(data, &pl)
+
+	// try to fix the plan in case of an error
+	if err != nil && pl.APIKey != nil {
+		if pl.Project.OrgID == "" {
+			return fmt.Errorf("failed to fix the plan with error: %w", err)
+		}
+
+		pl.APIKey["orgID"] = pl.Project.OrgID
+	}
+
+	// fix camelcase for "orgID" in a map
+	if _, ok := pl.APIKey["orgId"]; ok {
```

Comment:
I don't think that this exists in the data

Reply:
Now I realise that this "orgId" in that specific place I've created myself from the initial old version (and added into tests so they were failing).
Removing this block since there is no use for it.